packages/fluentui/react-northstar/src/components/Popup/Popup.tsx

```diff
@@ -505,6 +506,8 @@ export const Popup: React.FC<PopupProps> &
     if (open) {
       // when new state 'open' === 'true', save the last focused element
       updateTriggerFocusableRef();
+
+      setVirtualParent(popupContentRef.current, triggerRef.current);
```

Comment:
do we need cleanup? If popupContent or trigger gets unmounted, won't the references stay and cause leaks?

Reply:
> do we need cleanup?

Yes, it should be done.